reg/drone/srv/air_data_computer/_.0.1.uavcan

```diff
@@ -0,0 +1,50 @@
+# A generic air data computer service.
+#
+# An air data computer service is generally a non-interactive publish-only service: when activated,
+# it keeps publishing its measurements at a fixed rate over several subjects until deactivated.
+# The activation/deactivation, if supported, is managed via the standard arming command service.
+# The data update rates and other parameters are controlled via the Register API using implementation-defined
+# register names.
+#
+# An air data computer whose arming state is SLEEP is allowed to cease all network activity.
+#
+# An air data computer whose arming state is DISARMED is recommended to behave as if its state was ARMED;
+# implementations are recommended to leverage this state to perform any necessary maintenance activities
+# such as calibration, self-heating, etc. The service should not report its own status as ARMED until said
+# maintenance activities have been completed. Therefore, the high-level controler (e.g., the flight
+# management unit) may delay the take-off until the service is ready.
+#
+# An air data computer whose arming state is ARMED is required to comply with the requirements set out below.
+#
+# An armed service should publish the following data subjects synchronously at the same configurable rate
+# which should not be lower than the specified limit. The measurements should be low-pass filtered
+# to avoid frequency aliasing effects.
+#
+#   PUBLISHED SUBJECT       TYPE                                                NOTE
+#   calibrated airspeed     reg.drone.phy.kinematics.trans.Velocity1VarTs
+#   true airspeed           reg.drone.phy.kinematics.trans.Velocity1VarTs       optional
+#   pressure altitude data  reg.drone.phy.kinematics.trans.LinearVarTs          assume ISA; NED frame (+down, -up)
+#   static air data         reg.drone.phy.thermodynamics.PressureTempVarTs      pressure and OAT
+#
+# Observe that there is no subject for indicated airspeed. This is because per the principles of service-oriented
+# design, the air data computer is responsible for applying the necessary transformations on its data to render it
```

Comment:
I'm mission some context here. Is the air data computer meant as a "sensor", meaning that if you have multiple airspeed sensors you also get multiple publications? Or is more of an "estimator", that fuses multiple airspeed readings and publishes an averaged estimate/the best instance?
Depending on the above, if it's a "sensor" message then I would rather make it publish IAS istead of CAS, and leave the calibration to the estimator. And consequently also leave away TAS, as it can't be estimated without knowing CAS. If it's an "estimator" I think publishing CAS and TAS is fine.